### PR TITLE
S02158. Francine Update Dockerfile on Dev pipeline

### PR DESCRIPTION
**Pull request by francine-oss**
# WELCOME TO DEVOPS-EASY-LEARNING PULL REQUEST SYSTEM 
# Description of the change

> Detailed description here

## Checks and balances

- [ ] Tests added and run
- [ ] (External) Interfaces documented
- [ ] Has security implications (describe below)

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing
functionality to not work as expected)

## Related stories, issues and pulls

> Links to Pivotal Tracker stories, JIRA tickets or other pull requests here

## Security considerations

> Describe any security impact this change may have

### CODE REVIEW

**Review by francine-oss — COMMENTED** [review]
good job francine

**Review by francine-oss — COMMENTED** [review]
good job

**Review by agueno — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,7 @@
-FROM tomcat:8.0-alpine
+FROM tomcat:8.0-jre7
 COPY /home/ansible/webapp.war  /usr/local/tomcat/webapps
-VOLUME /volume
-VOLUME /gueno3
```
Comment (francine-oss) [inline Dockerfile:4]: good

**Review by linsay1999 — APPROVED** [review]

**Comment by igor581** [comment]
good job

**Review by RKemvou — APPROVED** [review]
got job francine

**Pull request closed without merging**